setup.py

```diff
@@ -2,7 +2,7 @@
 from setuptools import setup
 
 
-__version__ = "2021.08.24"
+__version__ = "2021.11.17"
```

Comment:
Is it necessary to use the date? Fine if yes, just checking.

Reply:
I'm good with any numbering convention. I believe I saw it in Qiita and just took the idea.

Reply:
ha! I meant to write the day; not date ... the format is fine, just not sure about including the day ... but perhaps we are expecting multiple version within the same month.

Reply:
Right, and if say a fix did need to be issued, it could just be 2021.11.1 or .2, etc. I'll fix that.